app/models/ability.rb

```diff
@@ -0,0 +1,41 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the user here. For example:
+    #
+    return unless user.present?
+
+    can :read, :all
+    return unless user.admin?
+
+    can :manage, :all
```

Comment:
- [ ] As requested by the previous reviewer please fix the problem below , this is happening when I am trying to see all posts you can please checkout [this link ](https://stackoverflow.com/questions/15855138/undefined-method-admin-for-nilnilclass) to see how it could help you

![image](https://user-images.githubusercontent.com/75126481/224242837-2112edef-1faf-4768-8bed-f88f2bf2b72d.png)

Reply:
@Krishnabot